scripts/prerequisites.sh

```diff
@@ -6,7 +6,7 @@ KIND_VERSION=0.11.1
 LINKERD_VERSION=stable-2.11.1
 KUBECTL_VERSION=v1.22.2
 MINIKUBE_VERSION=v1.23.2
-VEIDEMANNCTL_VERSION=0.4.0
+VEIDEMANNCTL_VERSION=0.5.1
```

Comment:
0.6.0 is the latest veidemannctl version: https://github.com/nlnwa/veidemannctl/releases/latest